_tools/fetch-sponsors.js

```diff
@@ -13,7 +13,7 @@
 //-----------------------------------------------------------------------------
 
 const fs = require("fs");
-const fetch = require("node-fetch");
+const fetch = require("node-fetch"); // eslint-disable-line node/no-unpublished-require
```

Comment:
I'm not sure why this is warning at the moment. Any ideas?

Reply:
`node-fetch` is declared at `devDependencies`, so the `eslint-plugin-node` reported it.

Reply:
That makes sense!

Reply:
I know this isn't the rule's current behavior, but it seems like it shouldn't report when the root package.json has `private: true`?

Reply:
@mysticatea Does that make sense? Maybe I'm missing something.

Reply:
can we disable these rules in eslintrc? (since it is not a node lib.)

Reply:
If you have `files: [...]` in your package.json, with publish files/dirs, that error should disappear.

Reply:
@aladdin-add’s suggestion makes sense to me. Will disable for the website.